backend/core/src/email/email.service.ts

```diff
@@ -20,15 +22,43 @@ import { Jurisdiction } from "../jurisdictions/entities/jurisdiction.entity"
 import { Language } from "../shared/types/language-enum"
 import { JurisdictionsService } from "../jurisdictions/services/jurisdictions.service"
 import { Translation } from "../translations/entities/translation.entity"
-import { IdName } from "../../types"
+import { IdName, ListingEventType, Unit } from "../../types"
 import { formatLocalDate } from "../shared/utils/format-local-date"
+import { formatCommunityType } from "../listings/helpers"
 
 type EmailAttachmentData = {
   data: string
   name: string
   type: string
 }
 
+const formatPricing = (values: number[]): string => {
+  const minPrice = Math.min(...values)
+  const maxPrice = Math.max(...values)
+  return `$${minPrice.toLocaleString()}${
+    minPrice !== maxPrice ? " - $" + maxPrice.toLocaleString() : ""
+  } per month`
+}
+
+const formatUnitDetails = (
+  units: Unit[],
+  field: string,
+  label: string,
+  pluralLabel?: string
+): string => {
+  const mappedField = units
```

Comment:
non-blocking suggestion: changing this to be a `reduce` instead of map and filter would reduce the number of times were iterating through units